web/src/queries/users.ts

```diff
@@ -83,12 +83,12 @@ const useFirstUserChanges = () => {
 
     return client.onEvent((event) => {
       if (event.type === "FirstUserChanged") {
-        const { fullName, userName, password, passwordEncrypted, autologin, data } = event;
+        const { fullName, userName, password, encryptedPassword, autologin, data } = event;
         queryClient.setQueryData(["users", "firstUser"], {
           fullName,
           userName,
           password,
-          passwordEncrypted,
+          encryptedPassword,
```

Comment:
I'm realizing that this might require a change in the backend side too :/

Reply:
I can simply storage "passwordEncrypted" from the backend as a "encryptedPassword" in the query cache. But would be nice... some consistency with this kind of things.

Reply:
Not only in the backend, but in the schema too. However, it should be pretty straightforward.

Reply:
The backend already uses `encryptedPassword`, that should be fine.

Reply:
> The backend already uses `encryptedPassword`, that should be fine.

Thus, previous code was broken, see 

https://github.com/agama-project/agama/blob/0094c4675e75a34ee49c80d9efe9c181a81a7d5b/web/src/queries/users.ts#L84-L94

Before the renaming I did, it was _extracting_ `passwordEncrypted`  from the event.

Reply:
Anyway, I have sent another commit changing all occurrences I've found. See https://github.com/agama-project/agama/pull/1787/commits/bf1099295deb711edb7830e0c265914726d26813